CHANGELOG.yaml

```diff
@@ -1,3 +1,7 @@
+master:
+  new features:
+    - Added $isoTimestamp to the dynamic variable list
```

Comment:
Let's add the GitHub issue ID here as well: https://github.com/postmanlabs/postman-app-support/issues/553